backend/shared/models/src/models/Member.ts

```diff
@@ -497,8 +498,18 @@ export class Member extends Model {
             }
 
             // Add the cheapest available membership
-            const cheapestMembership = defaultMemberships.sort(({membership: a, registration: ar}, {membership: b, registration: br}) => {
-                const diff = a.getPrice(platform.periodId, now)!.price - b.getPrice(platform.periodId, now)!.price
+            const defaultMembershipsWithOranization = await Promise.all(defaultMemberships.map(async ({membership, registration}) => {
+                const organizationId = registration.organizationId;
+                const organization = await Organization.getByID(organizationId);
```

Comment:
Nooit SQL queries in een loop als het niet nodig is. Gebruik dan één query lookup `await Organization.getByIDs`
-> risico dat je dezelfde organization meerdere keren ophaalt
-> je stuurt mogelijks meerdere queries naar de MySQL server, terwijl het in één query kan -> onnodige belasting